src/libsyntax/codemap.rs

```diff
@@ -799,7 +799,7 @@ impl CodeMap {
 
         let lo = self.lookup_char_pos_adj(sp.lo);
         let hi = self.lookup_char_pos_adj(sp.hi);
-        return (format!("{}:{}:{}: {}:{}",
+        return (format!("{}: {}:{} -> {}:{}",
```

Comment:
I would not use `->` since we use it for function types/signatures. I'd use `to` or `...` or keep the `:`